gremerror/networkerror.go

```diff
@@ -26,15 +26,13 @@ import "strconv"
 type NetworkError struct {
 	statusCode int
 	msg        string
-	origMsg    string
```

Comment:
Changes in this file were removed [here](https://gitlab.com/kaleidoscope.root/grammes-gremlin/-/commit/58aecaa0988a60f068717bef400c984533914001)

should be put back imo